Survey/Sources/Presentation/Modules/SurveyDetail/SurveyDetailView.swift

```diff
@@ -53,7 +39,9 @@ struct SurveyDetailView: View {
 
                         SButtonView(
                             isValid: .constant(true),
-                            action: {},
+                            action: {
+                                nextToSurveyQuestionTrigger.send()
```

Comment:
The variable is confused here. Should it be startSurveyTrigger? Correct me if I'm wrong, thanks. 
```suggestion
                                startSurveyTrigger.send()
```